include/clover_field.h

```diff
@@ -133,6 +133,8 @@ namespace quda {
       inverse(param.inverse),
       clover(param.clover),
       cloverInv(param.cloverInv),
+      csw(param.csw),
+      coeff(param.coeff),
```

Comment:
What does this `coeff` do?

Reply:
This is set to `kappa*csw`. in `loadCloverQuda()` and later used in `computeClover()`.